snippets/log6x-elasticsearch-unsupported-snip.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following assemblies:
+// observability/logging/cluster-logging-deploying.adoc
+//observability/logging/cluster-logging.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+In {product-title} 4.16, the Elasticsearch Operator is only supported for ServiceMesh, Tracing, and Kiali. This Operator is planned for removal from the OpenShift Operator Catalog in November 2025. The reason for removal is that the Elasticsearch Operator is no longer supported for log storage, and Kibana is no longer supported in {product-title} 4.16 and later versions. For more information on lifecycle dates, see link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Comment:
```suggestion
In {product-title} 4.16, the Elasticsearch Operator is only supported for ServiceMesh, Tracing, and Kiali. This Operator is planned for removal from the OpenShift Operator Catalog at a future release. The reason for eventual removal is that the Elasticsearch Operator is no longer supported for log storage, and Kibana is no longer supported in {product-title} 4.16 and later versions. For more information on lifecycle dates, see link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Reply:
Unless there's a great reason to add a specific month/year, like it meets the exception criteria here: https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future, I think we should keep this "future" statement more vague and less exact to avoid under-delivering on promises.

Reply:
I see in the comments screencap there is a specific date listed:
![image](https://github.com/user-attachments/assets/2a7f83a6-7266-4262-bd6c-01533fbf6f72)